### DISCUSSION TITLE
When will inkcanvas be supported?

### DISCUSSION THREAD

**Discussion by duwenlong2** [microsoft/WindowsAppSDK#4276] [Q&A]
On today's high refresh rate displays. WPF's inkcanvas is completely inadequate. He has a high delay.
The shape of the UWP pen is very closed. Cannot be adjusted. Currently, there are no excellent inkcanvas available for use.
In the WinUI3 product roadmap of this document, I often open it to check the progress of inkcanvas, but it has been several years and there has been no news at all.
If so, in which areas will WinUI3's inkcanvas be improved and optimized, and which features will be made available?

thank

**Comment by ChaimCL** [comment]
You can look this.https://github.com/microsoft/WindowsAppSDK/discussions/4256
